src/Umbraco.Web/Models/ContentEditing/ContentTypeBasic.cs

```diff
@@ -48,7 +48,7 @@ public string IconFilePath
             {
                 return IconIsClass
                            ? string.Empty
-                           : IOHelper.ResolveUrl("~/umbraco/images/umbraco/" + Icon);
+                           : IOHelper.ResolveUrl("images/umbraco/" + Icon);

```

Comment:
Basically like this: GlobalSettings.Path.EnsureEndsWith("/") + Icon; Since GlobalSettings.Path has already resolved the full path for the configured ~/Umbraco path.